CoreGraphFile/src/au/gov/asd/tac/constellation/graph/file/open/RecentFiles.java

```diff
@@ -354,9 +354,12 @@ protected static String convertFile2Path(final FileObject fo) {
     }
 
     public static FileObject convertPath2File(final String path) {
-        File f = new File(path);
-        f = FileUtil.normalizeFile(f);
-        return f == null ? null : FileUtil.toFileObject(f);
+        final File f = new File(path);
+        if (FileUtil.normalizeFile(f) == null || f.isDirectory()) {
+            return null;
+        } else {
+            return FileUtil.toFileObject(f);
```

Comment:
just noting what this code is replacing, should this function parse `f` or `FileUtil.normaliseFile(f)`?

Reply:
Valid point. We are only really adding the directory check and f may have been modified in FileUtil.normaliseFile(f). Will amend and recommit.